pype/hosts/blender/plugin.py

```diff
@@ -20,6 +20,36 @@ def asset_name(
     return name
 
 
+def asset_namespace(
+    asset: str, subset: str
+) -> str:
+    """Return a unique namespace based on the asset name."""
+    avalon_containers = [
+        c for c in bpy.data.collections 
+        if c.name == 'AVALON_CONTAINERS'
+    ]
+    loaded_assets = []
+    for c in avalon_containers:
+        loaded_assets.extend(c.children)
+    if avalon_containers is None:
+        return "1"
+    collections_names = [
+        c.name for c in loaded_assets 
+    ]
+    count = 1
+    name = f"{asset_name(asset, subset, str(count))}_CON"
+    while name in collections_names:
+        count += 1
+        name = f"{asset_name(asset, subset, str(count))}_CON"
+    return str(count)
```

Comment:
Only number may be confusing in scene inventory, it would be good to have asset name at the beginning of namespace and two digits number (like in maya) `return f"{asset}_{count:0>2}"`

Reply:
I can do that for the assets themselves, but the method to create the container in `AVALON_CONTAINERS` is in avalon-core!

Reply:
But here you define value for these, right?
![image](https://user-images.githubusercontent.com/43494761/86106482-e663aa80-bac0-11ea-8675-ed65ecf37cc0.png)

If you'll replace `return str(count)` with `return f"{asset}_{count:0>2}"` there will be `sally_01`, `sally_02`, `sally_03`, etc. instead of `1`, `2`, `3`, or?

Reply:
This is what happens:
![Annotation 2020-06-30 101515](https://user-images.githubusercontent.com/1087869/86108115-ac8fa580-baba-11ea-9761-a6c607d9e8a8.png)
![Annotation 2020-06-30 101530](https://user-images.githubusercontent.com/1087869/86108121-ad283c00-baba-11ea-9a67-cc2e574a60b0.png)

The asset will be changed using the double digits, but the container in `AVALON_CONTAINERS` will not, because it is done in avalon-core! In `load_rig.py`, line 117 we call `containerise_existing` (from avalon-core) which creates the container there!

Reply:
I'm going to leave it single digit for now, because the inconsistency with the collections in `AVALON_CONTAINERS` breaks the rest of the code.

Reply:
Aah, it's because of `~/avalon/blender/pipeline.py` in `parse_container`.
There is
```
data["objectName"] = container.name
```
I don't see connection with this PR, in that case this conversation is resolved. But can you create new PR in avalon-core to set `data["objectName"]` to not contain only number to match other hosts?

Reply:
I've made a new commit to support double digits, and made the PR in avalon-core: [#160](https://github.com/pypeclub/avalon-core/pull/160)!